packages/perseus/src/widgets/interactive-graphs/reducer/interactive-graph-reducer.ts

```diff
@@ -150,10 +150,7 @@ function doFocusPoint(
     switch (state.type) {
```

Comment:
Hmm it seems like my settings resulted in removing some white space from this file. I will revert that as there are no other changes.